cni-plugin/pkg/install/install.go

```diff
@@ -447,6 +439,11 @@ func copyFileAndPermissions(src, dst string) (err error) {
 		return fmt.Errorf("failed to rename file: %s", err)
 	}
 
+	if runtime.GOOS == "windows" {
+		// chmod doesn't work on windows
```

Comment:
Should we include a log of some sort here so that we have an idea why it quit early?

Reply:
Good point, will do!